Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeUtil.java

```diff
@@ -364,7 +364,7 @@ private static boolean isSystemOsAccount(String accountAddr) {
                     correlationAttrs.addAll(makeCorrAttrsFromCommunicationArtifact(artifact, attributes));
                 }
             } catch (CorrelationAttributeNormalizationException ex) {
-                logger.log(Level.SEVERE, String.format("Error normalizing correlation attribute (%s)", artifact), ex); // NON-NLS
+                logger.log(Level.WARNING, String.format("Error normalizing correlation attribute (%s): %s", artifact, ex.getMessage())); // NON-NLS
```

Comment:
For consistency, I think the InvalidAccountIDException below this should also be a warning.

Reply:
Sounds good.